bin/compile

```diff
@@ -107,3 +108,11 @@ fi
 
 # Set up the environment for runtimes now that compilation has finished
 echo 'export PATH=$HOME/.texlive/bin/x86_64-linux:$PATH' >> $PROFILE_D
+
+# run custom job
+if [ -f $BUILD_DIR/texlive.run ]; then
```

Comment:
I don't like the `.run`-extension. Heroku doesn't do that in their own build packs, so i'd like to keep it consistent. No extension should be fine.
Maybe you this should go into the `bin/` folder as it is done for the python buildpack?